packages/application-config/src/transformers.ts

```diff
@@ -44,7 +51,10 @@ const getPermissions = (
     appConfig.additionalOAuthScopes?.map(({ name }) => name) || [];
 
   const permissionKeys = entryPointUriPathToResourceAccesses(
-    appConfig.entryPointUriPath,
+    (appConfig as JSONSchemaForCustomApplicationConfigurationFiles)
+      .entryPointUriPath ||
+      (appConfig as JSONSchemaForCustomViewConfigurationFiles).env.production
```

Comment:
Permissions are calculated based on ID for Custom Views, as they don't have an entry point.
For a Custom View with the ID `123456`, the permissions keys would be `view123456` and `manage123456`.

Reply:
I think I don't follow the change you would like to introduce. 
Perhaps we can sync about this one.